TSG/ArcVMs/security-vulnerability-cve-2024-49060.md

```diff
@@ -0,0 +1,147 @@
+# ISO vulnerability for Arc VMs running on Azure Stack HCI, version 23H2
+
+## Description
+
+Microsoft has identified a security vulnerability that could expose the local admin credentials used during the creation of Arc VMs on Azure Stack HCI to non-admin users on the VM and on the hosts.  
+
+## Affected releases
+
+Arc VMs running releases prior to Azure Stack HCI, 2411 release are vulnerable.
+
+## Mitigation
+
+To mitigate this risk, we recommend that you rotate the passwords on the accounts used when creating Arc VMs on Azure Stack HCI. Completing this action will immediately protect your environment.  
+
+## Ongoing protection
+
+To ensure ongoing protection for newly provisioned VMs, update to the latest 2411 release. This release includes a fix that prevents this vulnerability for newly provisioned VMs, eliminating the need for additional password rotation.  
+
+To mitigate this vulnerability, first identify all the Arc VMs created before the Azure Stack HCI, 2411 release. Then, change the account password on each VM.
+
+### Identify Arc VMs
+
+1. Identify all the Arc VMs created before the Azure Stack HCI, 2411 release, run the following script. Make sure to change the variables in the beginning with the appropriate values.
```

Comment:
"the variables in the beginning" to "the variables in the beginning of the script"